lib/components/util/span-with-space.js

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components'
+
+const SpanWithSpace = styled.span`
+  margin-right: ${props => props.margin}em
```

Comment:
`margin-right` will not work with right-to-left languages (but using the `::after` pseudo-element will).

Reply:
Changed to use `::after` in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/2ccefb263ad8863edab72e9d796596beda7336f0